src/pudl/transform/eia861.py

```diff
@@ -593,8 +603,17 @@ def _tidy_class_dfs(df, df_name, idx_cols, class_list, class_type, keep_totals=F
     # Separate customer classes and reported data into a hierarchical index
     logger.debug(f"Stacking EIA861 {df_name} data columns by {class_type}.")
     data_cols = _filter_class_cols(raw_df, class_list)
+
+    # Create a regex identifier that splits the column headers based on the strings
+    # deliniated in the class_list not just an underscore. This enables prefixes with
+    # underscores such as fuel_cell as opposed to single-word prefixes followed by
+    # underscores. Final string looks like: '(?<=customer_test)_|(?<=unbundled)_'
+    # This ensures that the underscore AFTER the desired string (that can also include underscores)
+    # is where the column headers are split, not just the first underscore.
+    class_list_regex = '|'.join(['(?<=' + col + ')_' for col in class_list])
```

Comment:
this is great! just one line for so much extra simplicity in the column names 👍

Reply:
Great! I'm going to change some stuff in the net_metering/non_net_metering files.